js/client-chat.js

```diff
@@ -1237,6 +1237,20 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'globalnotify':
+					app.rooms[''].addPM('~', '/raw ' + row.slice(1).join('|'), '' + this.users[app.user.changed.userid]);
+					if (!Tools.prefs('mute') && Tools.prefs('notifvolume')) {
+						soundManager.getSoundById('notif').setVolume(Tools.prefs('notifvolume')).play();
+					}
+					break;
+
+				case 'notify':
+					if (!Tools.prefs('mute') && Tools.prefs('notifvolume')) {
+						soundManager.getSoundById('notif').setVolume(Tools.prefs('notifvolume')).play();
+					}
+					this.notifyOnce("You have a notification", "\"" + row.slice(1).join('|') + "\"", 'highlight');
```

Comment:
Yeah i seperated notify and globalnotify since they function differently now. One Hls you in a room, the other dosent HL in rooms. 

Currently when /declare is used, it sends `|notify|${Chat.escapeHTML(target)}` to the client, the client then takes whatever is after `|notify|` and puts that as the notification reason. I'm not sure what you mean by "pass both the title and message into `|notify|`"